docs/getting-started-guides/logging.md

```diff
@@ -1,17 +1,206 @@
-## Logging
+# Logging
 
-**Experimental work in progress.**
+A Kubernetes cluster will typically be humming along running many system and application pods. How does the system administrator collect, manage and query the logs of the system pods? How does a user query the logs of their application which is composed of many pods which may be restarted or automatically generated by the Kubernetes system? These questions are addressed by the Kubernetes **cluster level logging** services.
 
-### Logging with Fluentd and Elastiscsearch
+Cluster level logging for Kubernetes allows us to collect logs which persist beyond the lifetime of the pod’s container images or the lifetime of the pod or even cluster. In this article we assume that a Kubernetes cluster has been created with cluster level logging support for sending logs to Google Cloud Logging. This is an option when creating a Google Container Engine (GKE) cluster, and is enabled by default for the open source Google Compute Engine (GCE) Kubernetes distribution. After a cluster has been created you will have a collection of system pods running that support monitoring, logging and DNS resolution for names of Kubernetes services:
 
-To enable logging of the stdout and stderr output of every Docker container in
+```
+$ kubectl get pods
+NAME                                           READY     REASON    RESTARTS   AGE
+fluentd-cloud-logging-kubernetes-minion-0f64   1/1       Running   0          32m
+fluentd-cloud-logging-kubernetes-minion-27gf   1/1       Running   0          32m
+fluentd-cloud-logging-kubernetes-minion-pk22   1/1       Running   0          31m
+fluentd-cloud-logging-kubernetes-minion-20ej   1/1       Running   0          31m
+kube-dns-v3-pk22                               3/3       Running   0          32m
+monitoring-heapster-v1-20ej                    0/1       Running   9          32m
+```
+
+Here is the same information in a picture which shows how the pods might be placed on specific nodes.
+
+![Cluster](../../examples/blog-logging/diagrams/cloud-logging.png)
+
+This diagram shows four nodes created on a GCE cluster with the name of each VM node on a purple background. The internal and public IPs of each node are shown on gray boxes and the pods running in each node are shown in green boxes. Each pod box shows the name of the pod and the namespace it runs in, the IP address of the pod and the images which are run as part of the pod’s execution. The default namespace is used to run privileged system services rather than user applications. Here we see that every node is running a fluentd-cloud-logging pod which is collecting the log output of the containers running on the same node and sending them to Google Cloud Logging. A pod which provides a DNS service runs on one of the nodes and a pod which provides monitoring support runs on another node.
```

Comment:
"The default namespace is used to run privileged system services rather than user applications"

Isn't the default namespace typically used for both?